packages/components/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## Unreleased
 
+### New Feature
+
+-   `CustomSelectControl`: Add `onMouseOver` and `onMouseOut` callback props to allow handling of these events by parent coomponents ([#44955](https://github.com/WordPress/gutenberg/pull/44955))
+-   `BoxControl`: Add `onMouseOver` and `onMouseOut` callback props to allow handling of these events by parent coomponents ([#44955](https://github.com/WordPress/gutenberg/pull/44955))
```

Comment:
```suggestion
-   `BoxControl` & `CustomSelectControl`: Add `onMouseOver` and `onMouseOut` callback props to allow handling of these events by parent components ([#44955](https://github.com/WordPress/gutenberg/pull/44955))
```

I think we can condense these two near-identical entries to make things a little more readable. Either way, we should correct the typo, `coomponents` to `components`.

Reply:
Thanks, fixed.